docs/channels/emails.rst

```diff
@@ -249,7 +287,7 @@ Mautic works most effectively with high send volumes if you use the queued deliv
 
 .. code-block:: shell
     
-    php /path/to/mautic/bin/console mautic:email:process
+    php /path/to/mautic/bin/console messenger:consume email_transport
```

Comment:
Please update also the cron jobs file, and the command line interfaces file. Both will need to have this command included and explained.